flatpak_builder_lint/checks/flathub_json.py

```diff
@@ -1,5 +1,7 @@
 from . import Check
 
+supported_arches = {"x86_64", "aarch64"}
```

Comment:
Does it make sense to make it a global variable? It could be moved to the class itself.

Reply:
actually it should be to some extent a global one, as we could check the same for the `only-arches` or `exclude-arches` in modules or sources.

Further down, we could have check with heuristic for errors like "you are using x86_64 binaries on aarch64".

I can't stop counting how often I have to mention this in reviews.